### PR TITLE
LogsVolume: Make log volume work with chunking

### PR DESCRIPTION
**Pull request by svennergr**
This PR makes the logs volume work with query chunking.

### CODE REVIEW

**Comment by svennergr** [comment]
@matyax @gabor currently it's not visible to the user once the logs volume histogram is done loading. So there's no indicator for a streaming state. We should improve that in the future.

**Comment by matyax** [comment]
> We should improve that in the future.

Gotcha. @svennergr would you mind creating an issue about that and making it part of https://github.com/grafana/grafana/issues/63174 ? Thank you! 🙇

File: public/app/core/logsModel.ts [hunk 1]
```diff
@@ -733,7 +722,24 @@ export function queryLogsVolume<TQuery extends DataQuery, TOptions extends DataS
...
+            aggregatedLogsVolume[0].meta = {
+              custom: {
+                targets: options.targets,
+                absoluteRange: { from: options.range.from.valueOf(), to: options.range.to.valueOf() },
+              },
+            };
+          }
+          rawLogsVolume = aggregatedLogsVolume;
+          observer.next({
+            state: LoadingState.Streaming,
```
Comment (matyax) [inline public/app/core/logsModel.ts:739]: Did you check if using `LoadingState.Loading` had any visible impact in the UI?

**Review by matyax — APPROVED** [review]
Looks good! GJ

File: public/app/core/logsModel.ts (hunk 1 above)
Comment (svennergr) [reply public/app/core/logsModel.ts:739]: Doesn't change anything, but in b82d606 I added a small indicator:

image

WDYT?

Comment (matyax) [reply public/app/core/logsModel.ts:739]: Wunderbar!

**Review by matyax — COMMENTED** [review]
Would you mind adding some testing coverage for these changes? In particular to the loading indicator, so 1) we don't get regressions and 2) we can use the test to explain why that code is there. Thanks!

File: public/app/features/explore/LogsVolumePanel.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export function LogsVolumePanel(props: Props) {
     }
   }
 
-  let extraInfo;
+  let extraInfo = <></>;
```
Comment (matyax) [inline public/app/features/explore/LogsVolumePanel.tsx:79]: I think you can use `null`?

**Review by matyax — APPROVED** [review]
👏 👏

**Comment by Elfo404** [comment]
Can we please add more info about what this PR does and why for future reference?

**Pull request merged**